bin/builder-core.js

```diff
@@ -38,6 +45,11 @@ module.exports = function (opts, callback) {
     runner: runner
   });
 
+  // Drain messages if any.
+  (opts.msgs || []).forEach(function (obj) {
```

Comment:
Isn't this 1:1 with your _drainQueue method? Also, how does this "Drain" if `opts.msgs` remains the same after this runs